707. Design Linked List

```diff
@@ -0,0 +1,53 @@
+class ListNode:
+    def __init__(self, val=0, nextNode=None):
+        self.val = val
+        self.next = nextNode
+
+        
+class MyLinkedList:
+    def __init__(self):
+        self.size = 0
+        self.head = ListNode()
+
+    def get(self, index: int) -> int:
+        if index >= self.size: return -1
+        curr = self.head
+        for _ in range(index):
+            curr = curr.next
+        return curr.next.val
+        
+    def addAtHead(self, val: int) -> None:
+        self.addAtIndex(0, val)
```

Comment:
It's great that you are using already implemented methods and not re-writing code for each case.